### DISCUSSION TITLE
Deployment Intune

### DISCUSSION THREAD

**Discussion by layellucas** [Romanitho/Winget-AutoUpdate#366] [Q&A]
I juste saw that this git is more UP to date than winget configurator for Intune.
Simple question, Can I use this Git to deploy winget update with admx on Intune ?

Or maybe I Can just deploy whitelist/Blacklist in registre Key "Uninstall" ? (Updated using remediation for exemple).
Thanks for your help.

**Comment by Romanitho** [comment]
in your context, just update your whitelist txt file (remediation if you want)
